broker-cli/commands/wallet.js

```diff
@@ -560,6 +560,10 @@ async function history (args, opts, logger) {
       style: { head: ['gray'] }
     })
 
+    if (json) {
+      return logger.info(JSON.stringify(transactions, null, 2))
```

Comment:
Same comment as above

Reply:
we should use `logger` here (we use logger in the underlying method as well)